### PR TITLE
Dashboard update Sunday

### PR DESCRIPTION
**Pull request by julianharr**
• Began more work on dashboard
• Nearly finished within a couple of hours, then lost all trace of scss file and had to restart
• Still work to be done

**UPDATE** 
• Overhauled dashboard with cleaner looking layout
• Completely responsive

https://user-images.githubusercontent.com/62631835/107143363-c9615500-6988-11eb-8ade-e33ae21fabb4.mov

### CODE REVIEW

File: app/views/pages/dashboard.html.erb [hunk 1]
```diff
@@ -1,30 +1,49 @@
...
+        <h4>Listing History</h4>
+      </div>
+    </div>
+    <div class="row border m-3">
+      <% @user.bikes.each do |bike| %>
+        <div class="card-product m-1">
+          <%= cloudinary_imgs(bike.images[0].key,bike) %>
+          <div class="card-product-infos d-flex flex-column justify-content-start">
+            <h2 class="card-title"> <%= bike.name %> </h2> <hr>
+            <p><i class="fas fa-dollar-sign"></i> <%= bike.id %></p> <hr>
```
Comment (matoni109) [inline app/views/pages/dashboard.html.erb]: We could use 

<%= bike.price %>

File: app/views/pages/dashboard.html.erb [hunk 2]
```diff
@@ -1,30 +1,49 @@
...
-    <div class="col-lg-3 col-md-4 col-6 card-booking-category">
-        <%= booking.till %>
-        <%= link_to 'Add a review', new_booking_review_path(booking) %>
-      </a>
+  </div>
+  <!-- Rental History -->
+    <div class="row d-flex justify-content-center p-3">
+    <div class="grid-card col-md-12 d-flex align-items-center p-3 m-4">
+      <i class="fas fa-list-alt mr-4"></i>
+      <h4>Rental History</h4>
```
Comment (matoni109) [inline app/views/pages/dashboard.html.erb:55]: Below could go something like below here ( but in a card )and the renters avatar ontop )  :   
https://uikit.lewagon.com/documentation#card_trip

Do we want users to have a public profile page ? like => https://www.spinlister.com/profile/18873

[ # user bookings card block sample :just outputs values no style ]
```
<!-- User's upcoming rentals -->
<div class="container my-5">
    <h3>Rentals</h3>
    <div class="row text-center text-lg-left">
        <% @user.bookings.each do |booking| %>
        <div class="col-lg-3 col-md-4 col-6 card-booking-category">
            <%= booking.bike.name %>
            <%= image_tag booking.bike.images[0], class:"img-fluid mr-4", style: "height: 150px; width: 150px; " %>
            <%   mytime_1 = booking.from %>
            <%  mytime_2 = booking.till %>
            <%= booking.from.strftime("%A #{mytime_1.day.ordinalize} %b %Y") %>
            <%= booking.till.strftime("%A #{mytime_2.day.ordinalize} %b %Y") %>
            <%= booking.total_price %> from User
            <%= booking.bike.user.first_name
 %>
            <%= image_tag booking.bike.user.avatar, class:"img-fluid mr-4", style: "height: 150px; width: 150px; border-radius: 50%;" %>
            <%= link_to 'Add a review', new_booking_review_path(booking) %>
        </div>
        <% end %>
    </div>
</div>
```

**Review by matoni109 — COMMENTED** [review]
Hi Guys,
some ideas for the Dashboard

File: app/views/pages/dashboard.html.erb (hunk 1 above)
Comment (julianharr) [reply app/views/pages/dashboard.html.erb]: Yep easy I'll pop it in

File: app/views/pages/dashboard.html.erb (hunk 2 above)
Comment (julianharr) [reply app/views/pages/dashboard.html.erb:55]: Nice Chris, I changed most of the look and data retrieval just then.

**Pull request merged**